services/server/server.go

```diff
@@ -331,6 +335,20 @@ func (s *Server) UpdateStatus(ctx context.Context, req *serverpb.UpdateStatusReq
 	}, nil
 }
 
+// StartSecurityChecks starts STT checks execution.
+func (s *Server) StartSecurityChecks(ctx context.Context, request *serverpb.StartSecurityChecksRequest) (*serverpb.StartSecurityChecksResponse, error) {
+	err := s.checksService.StartChecks(ctx)
+	if err != nil {
+		if err == services.STTDisabledError {
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
err113: do not compare errors directly, use errors.Is() instead: "err == services.STTDisabledError" (goerr113)